tests/test_r2s.py

```diff
@@ -546,28 +559,45 @@ def test_r2s_script_step_by_step():
         thisdir, "files_test_r2s", "r2s_examples", "unstructured_r2s_run")
     _r2s_test_step1(r2s_run_dir, remove_step1_out)
     _r2s_test_step2(r2s_run_dir, remove_step1_out)
-
+    # test openmc r2s
+    try:
+        import openmc
+    except:
+        raise SkipTest
+    r2s_run_dir = os.path.join(
+            thisdir, "files_test_r2s", "r2s_examples", "openmc_r2s")
+    _r2s_test_step1(r2s_run_dir, remove_step1_out)
+    _r2s_test_step2(r2s_run_dir, remove_step1_out)
 
 def test_r2s_script():
+    # skip test without dagmc
+    try:
+        from pyne import dagmc
+    except ImportError:
+        raise SkipTest
 
-   # skip test without dagmc
-   try:
-       from pyne import dagmc
-   except ImportError:
-       raise SkipTest
-
-   remove_step1_out = False
-   r2s_run_dir = os.path.join(
-       thisdir, "files_test_r2s", "r2s_examples", "r2s_run")
-   _r2s_test_step1(r2s_run_dir, remove_step1_out)
-   _r2s_test_step2(r2s_run_dir, remove_step1_out)
-   # test sub-voxel r2s
-   r2s_run_dir = os.path.join(
-       thisdir, "files_test_r2s", "r2s_examples", "subvoxel_r2s_run")
-   _r2s_test_step1(r2s_run_dir, remove_step1_out)
-   _r2s_test_step2(r2s_run_dir, remove_step1_out)
-   # test unstructured r2s
-   r2s_run_dir = os.path.join(
-       thisdir, "files_test_r2s", "r2s_examples", "unstructured_r2s_run")
-   _r2s_test_step1(r2s_run_dir, remove_step1_out)
-   _r2s_test_step2(r2s_run_dir, remove_step1_out)
+    remove_step1_out = False
+    # test voxel r2s
+    r2s_run_dir = os.path.join(
+        thisdir, "files_test_r2s", "r2s_examples", "r2s_run")
+    _r2s_test_step1(r2s_run_dir, remove_step1_out)
+    _r2s_test_step2(r2s_run_dir, remove_step1_out)
+    # test sub-voxel r2s
+    r2s_run_dir = os.path.join(
+        thisdir, "files_test_r2s", "r2s_examples", "subvoxel_r2s_run")
+    _r2s_test_step1(r2s_run_dir, remove_step1_out)
+    _r2s_test_step2(r2s_run_dir, remove_step1_out)
+    # test unstructured r2s
+    r2s_run_dir = os.path.join(
+        thisdir, "files_test_r2s", "r2s_examples", "unstructured_r2s_run")
+    _r2s_test_step1(r2s_run_dir, remove_step1_out)
+    _r2s_test_step2(r2s_run_dir, remove_step1_out)
+    # test openmc r2s
+    try:
+        import openmc
+    except:
```

Comment:
We'll want to make sure the exception is an `ImportError` here.
```suggestion
    except ImportError:
```